File: laboratorios/lab02/l-javascript/clases/Estudiante.js
import { Persona } from "./Persona.js"

export class Estudiante extends Persona {
  constructor(id, nombre, edad) {
    super(id, nombre, edad)
  }

  toString() {
    return `Estudiante{id='${this.id}', nombre='${this.nombre}', edad=${this.edad}}`
  }
}

File: laboratorios/lab02/l-java/src/main/java/com/tuorg/poo/clases/Estudiante.java
package com.tuorg.poo.clases;

//Hereda de Persona
public class Estudiante extends Persona {
    public Estudiante(String id, String nombre, int edad) {
        super(id, nombre, edad);
    }

    @Override
    public String toString() {
        return "Estudiante{" +
                "id='" + getId() + '\'' +
                ", nombre='" + getNombre() + '\'' +
                ", edad=" + getEdad() +
                '}';
    }
}